src/System.Security.Cryptography.Cng/src/Microsoft/Win32/SafeHandles/NCryptSafeHandles.cs

```diff
@@ -194,17 +211,18 @@ private enum OwnershipState {
         ///     See code:Microsoft.Win32.SafeHandles.SafeNCryptHandle#NCryptHandleDuplicationAlgorithm for
         ///     details about the algorithm.
         /// </summary>
-        private T DuplicateDuplicatedHandle<T>() where T : SafeNCryptHandle, new() {
-            Contract.Ensures(m_ownershipState == OwnershipState.Duplicate);
+        private T DuplicateDuplicatedHandle<T>() where T : SafeNCryptHandle, new()
```

Comment:
Jeremy ported the safe handle code, best to ask him on this one. I only made changes to fit the coding style rules.